include/riak_kv_ts.hrl

```diff
@@ -134,4 +134,10 @@
             'EXPLAIN' = ?SQL_SELECT{} :: ?SQL_SELECT{}
         }).
 
+-record(riak_sql_delete_query_v1,
+        {
+          'FROM'     = <<>> :: binary() | {list, [binary()]} | {regex, list()},
+          'WHERE'    = []   :: [riak_ql_ddl:filter()],
```

Comment:
`'WHERE'` _must not_ be an empty list here, no? This applies to `#riak_sql_select_*` records as well, even though it may be conceivably relaxed for those in future. But it still definitely is mandatory for `#riak_sql_delete_query`.